### PR TITLE
feat(docs): document erc721

### LINKED ISSUE #213: [Docs]: Erc721
add Erc721 antora documentation guide

### PR DESCRIPTION
**Pull request by qalisander**
Resolves #213

#### PR Checklist

- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by bidzyyys** [comment]
Is it ready @qalisander ?

File: docs/modules/ROOT/pages/erc721.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        self._next_token_id.set(token_id);
+
+        self.erc721._mint(player, token_id)?;
+        self.uri_storage._set_token_uri(token_id, token_uri);
+
+        Ok(token_id)
+    }
+
+    #[selector(name = "tokenURI")]
+    pub fn token_uri(&self, token_id: U256) -> Result<String, Vec<u8>> {
```
Comment (qalisander) [inline docs/modules/ROOT/pages/erc721.adoc:54]: I've noticed that the example similar to solidity one looks complex for documentation

**Review by alexfertel — APPROVED** [review]
LGTM!

**Review by bidzyyys — APPROVED** [review]
LGTM!

**Pull request merged**